packages/back-end/src/api/openapi/schemas/ExperimentResultsRow.yaml

```diff
@@ -0,0 +1,157 @@
+type: object
+required:
```

Comment:
Should we add a statsEngine field instead of relying on certain fields like pValue being null as the only way to differentiate?